alternative_error_messages.py

```diff
@@ -0,0 +1,44 @@
+from random import random as random
+
+from flask import session
+from hedy_content import ALTERNATIVE_ERROR_LANGUAGES
+from website.auth import (current_user, update_test_group_in_session)
```

Comment:
Not sure the `()` are needed, could you remove them to be consisted with our code base?

Reply:
done!